Cms_frontend/src/pages/Editor.tsx

```diff
@@ -10,11 +11,16 @@ const Container = styled.div`
 `
 const Editor: React.FC = () => {
     return (
-        <div className="Editor">
+        <div style={{ height: '100%' }}>
```

Comment:
another way that might be helpful is instead of using inline styles, we could import `{ Box }` from material ui and then just write `<Box height="100%">` , just something i personally like to do nowadays instead of using inline style tags